src/components/grid-aware/Covid19InfoBoxBlock/Covid19InfoBoxBlock.module.css

```diff
@@ -0,0 +1,30 @@
+.bleedWrapper {
+  background: var(--color-gray-300);
+  display: grid;
+  grid-template-columns: 1fr minmax(var(--grid-content-min-width), var(--grid-content-max-width)) 1fr;
+  margin: auto;
+  max-width: var(--grid-bleed-max-width);
+  padding: 60px;
+}
+
+@media (--tablet-and-down) {
+  .bleedWrapper {
+    display: block;
+    padding: 50px 40px;
+  }
+}
+
+.bleedMainContent {
+  grid-column: 2;
```

Comment:
<img width="592" alt="Screen Shot 2020-11-15 at 3 08 46 PM" src="https://user-images.githubusercontent.com/30575095/99199515-94ac7b80-2754-11eb-831f-52f4bb3c0c27.png">

I see a right bleed here. I don't think this component is just in column 2 but extends also to touch column 3